tests/manual-test-cases/Group5-Functional-Tests/5-24-Non-vSphere-Local-Cluster.robot

```diff
@@ -34,7 +34,9 @@ Non vSphere Local Cluster Install Setup
     Set Suite Variable  @{list}  @{esx_names}[0]  @{esx_names}[1]  @{esx_names}[2]  %{NIMBUS_USER}-${vc}
 
     # Finish vCenter deploy
-    ${output}=  Wait For Process  ${pid}
+    ${output}=  Wait For Process  ${pid}  timeout=40 minutes  on_timeout=terminate
```

Comment:
lgtm, but I would bump this to probably 70 minutes I think, I have seen working installs take 40-50 minutes at heavily loaded times.